django_afip/models.py

```diff
@@ -1708,3 +2008,55 @@ def __repr__(self) -> str:
     class Meta:
         verbose_name = _("receipt validation")
         verbose_name_plural = _("receipt validations")
+
+
+class CaeaCounter(models.Model):
```

Comment:
I think it could be, but I thought that in this way is more clear and it was my first thought.

Reply:
There's chances of things running out-of-sync when you keep pre-calculated values, and it becomes a burden to always have to remember to keep it in sync.

It's usually best to avoid because you eliminate the risk of it falling out-of-sync. Plus, it should be simple to fetch the largest number from the DB. Something like:

```py
Receipts.objects.filter(
    point_of_sales=some_pos,
    receipt_type=some_type
).order_by(
    "-receipt_number"
).values_list(
    "receipt_number,
    flat=True
).first()
```

(untested, but should return just the number)

Reply:
This was my fear, I thought that a counter could help. I will write some tests and work around to see if there is any problem

Reply:
Solved in: 32bc979.

If a company has used caea in the past and for some reason in the DB we have not record of that `PointOfSales` and `ReceiptType`, we could get a Error from afip where we didn't assing the correct number. This need to be ensure for us o for the developer who use this?